### PR TITLE
added product listing page

### PR DESCRIPTION
**Pull request by vidhi-thakur**
added product listing page with vertical cards and a filter component
image

check preview URL [here](https://deploy-preview-6--savvy-store.netlify.app/pages/product.html)

### CODE REVIEW

**Review by Rockyraven — APPROVED** [review]
code is clean 
everything looks goods to me 
you are good to go

**Review by shuklahimanshu81 — APPROVED** [review]
Everything looks good. Go ahead to merge.

File: page-css/product.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: var(--lg-spacing);
+}
+
+@media screen and (max-width: 950px) {
+  .product-listing {
+    grid-template-columns: 1fr;
+    min-height: 88.5vh;
```
Comment (msdeshmukh009) [inline page-css/product.css:38]: I think you have mentioned `min-height: 88.5vh;` before as well. So no need to do it again here.

**Comment by vidhi-thakur** [comment]
> code is clean everything looks goods to me you are good to go

thank you

**Comment by vidhi-thakur** [comment]
> Everything looks good. Go ahead to merge.

thanks 👍

**Review by msdeshmukh009 — APPROVED** [review]

File: page-css/product.css (hunk 1 above)
Comment (vidhi-thakur) [reply page-css/product.css:38]: thanks for letting me know, updated the code

**Pull request merged**